examples/notebooks/healthcare/beam_post_hl7_messages_to_hcapi.ipynb

```diff
@@ -0,0 +1,484 @@
+      "source": [
+        "Lets install necessary packages"
+      ],
+      "metadata": {
+        "id": "81wCK9XnS6Sc"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install apache-beam[gcp]"
+      ],
+      "metadata": {
+        "id": "Yv1phmRZS23c"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "**GCP Setup**\n",
+        "1. Authenticate your notebook by `gcloud auth application-default login` in the Colab terminal.\n",
+        "\n",
+        "2. Run `gcloud config set project <YOUR-PROJECT>`\n",
+        "\n",
+        "Set the variables in the next cell based upon your project and preferences. The files referred to in this notebook nlpsample*.csv are in the format with one\n",
+        "blurb of clinical note.\n",
+        "\n",
+        "Note that below, **us-central1** is hardcoded as the location. This is because of the limited number of [locations](https://cloud.google.com/healthcare-api/docs/how-tos/hl7v2-messages) the API currently supports."
+      ],
+      "metadata": {
+        "id": "tpePe_yOsdSJ"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Before running please set the following variables as arguments as mentioned below\n"
+      ],
+      "metadata": {
+        "id": "_1Q3mw1usnoE"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "args = {'gcp_project':'xxx', #GCP project ID\n",
+        "        'gcp_region':'xxx', # GCP project region\n",
+        "        'temp_location':'gs://<YOUR Bucket>/tmp', #input location where your HL7 messages are stored in GCS bucket\n",
+        "        'input_file':'gs://<YOUR Bucket>/my_message.hl7', #input location where your HL7 messages are stored in GCS bucket\n",
+        "        'hcapi_project_id':'xxxxxx', #healthcare API project ID\n",
+        "        'hcapi_dataset':'xxxx', #healthcare dataset\n",
+        "        'hcapi_version':'v1', #healthcare API version by defualt v1\n",
+        "        'hcapi_location':'xxxx', #healthcare API configured location\n",
+        "        'hcapi_hl7_store':'xxx', #healthcare api hl7 store\n",
+        "        'hcapi_fhir_store':''}"
+      ],
+      "metadata": {
+        "id": "a722GbqdvgOX"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "**HCAPI Utils class**\n",
+        "\n",
+        "Below is the code snippet which describes class having healthcare API conncetions and configurations. Basic functionality, includes constructing the hcapi_url as per the input parameters, clean the HL7 message in terms of proper formatting and post hl7v2 message to hl7v2 store. You can add more transformations as per your requirements."
+      ],
+      "metadata": {
+        "id": "NHzk8JIqxQoa"
+      }
+    },
+    {
+      "cell_type": "code",
```

Comment:
The following code blocks have a lot of code but very little output. This may not be a great user experience. Is it possible to break these into multiple cells, and provide some output so the user could see what is being run, and why it's significant?

Reply:
Some sort of system diagram may be useful here to convey the point